pkgs/top-level/all-packages.nix

```diff
@@ -25866,6 +25866,8 @@ in
 
   zathura = callPackage ../applications/misc/zathura { };
 
+  zerobin = python3Packages.callPackage ../applications/networking/zerobin { };
```

Comment:
There's this rule of thumb that recommends using in this case:

```nix
  zerobin = callPackage ../.... { };
```

And use `python3Packages.buildPythonApplication` inside the expression.

Reply:
@doronbehar I've changed as you suggested, and rebased after changes from a6ce00c50c36624fec06b2b756a766d4d0f4a888